src/pages/receiving/components/EditModalTable.ts

```diff
@@ -35,6 +35,10 @@ class Row extends BasePageModel {
     this.expiryDatePickerField = new DatePicker(page, 'Expiry', row);
     this.quantityShippedField = new TextField(page, 'Quantity shipped', row);
   }
+
+  get quantityShipped() {
+    return this.row.getByRole('spinbutton');
```

Comment:
What if the will be a second quantity input with spinbutton role on that page? I think we should improve the accessor here